src/main/java/com/SAS/game_event_logger/GameEventLogger.java

```diff
@@ -0,0 +1,27 @@
+package com.SAS.game_event_logger;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class GameEventLogger {
+    private String gameID;
```

Comment:
It should be the Game object instead of the gameID.

Reply:
but then we have Game that poing he's logger and logger that point it's Game, it can cause Bugs cuz of loops in Pointers.